File: app/src/main/java/se/gustavkarlsson/aurora_notifier/android/gui/activities/settings/SettingsActivity.java
package se.gustavkarlsson.aurora_notifier.android.gui.activities.settings;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.util.Log;

public class SettingsActivity extends AppCompatActivity {
	private static final String TAG = SettingsActivity.class.getSimpleName();

	@Override
	protected void onCreate(Bundle savedInstanceState) {
		Log.v(TAG, "onCreate");
		super.onCreate(savedInstanceState);

		getFragmentManager().beginTransaction()
				.replace(android.R.id.content, new SettingsFragment())
				.commit();
	}
}

File: app/src/main/kotlin/se/gustavkarlsson/skylight/android/gui/activities/settings/SettingsActivity.kt
package se.gustavkarlsson.skylight.android.gui.activities.settings

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import se.gustavkarlsson.skylight.android.Skylight
import se.gustavkarlsson.skylight.android.dagger.components.SettingsActivityComponent

class SettingsActivity : AppCompatActivity() {

	lateinit var component: SettingsActivityComponent
		private set

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
		component = Skylight.applicationComponent.getSettingsActivityComponent()
        fragmentManager.beginTransaction()
                .replace(android.R.id.content, SettingsFragment())
                .commit()
    }
}
